tools/generate-test-files.sh

```diff
@@ -1,14 +1,56 @@
 #!/bin/bash
-# ASSUMES YOU'VE CHECKED OUT THE pypfb repo and are in the tools directory of java-pfb
+# Script: java-pfb/tools/generate-test-files.sh
```

Comment:
maybe this already exists somewhere in a read me that I missed (if so, reference to it would be helpful) but just a quick summary of what this script does and how to run would be helpful! Looks like it should be run as "java-pfb/tools/generate-test-files.sh <PathtoAvroFile>"?

Reply:
Added an explanatory comment in 3532d36328dd8d7fcdadac73462fc23602b19536